bitrise.yml

```diff
@@ -428,7 +428,7 @@ workflows:
 
             export XCODE_XCCONFIG_FILE=/tmp/tmp.xcconfig
 
-            envman add --key XCODE_XCCONFIG_FILE --value /tmp/tmp.xcconfig 
+            envman add --key XCODE_XCCONFIG_FILE --value /tmp/tmp.xcconfig
```

Comment:
@dnarcese I am seeing some extra changes in the file that are not related to v31.x branch, are these changes really required for BR? 🤔

Reply:
They should all just be whitespace changes, right?

Reply:
Yeah just whitespace

Reply:
Those were auto changed from Bitrise. They won't affect anything so I figured it was easier to leave them then go through and undo them all

Reply:
Oh so this was edited from BR and you just downloaded the file and are now updating it in the repo right?

Reply:
yes